std/assembly/bindings/DOM.ts

```diff
@@ -0,0 +1,171 @@
+@extern
+export declare class Object {
```

Comment:
How about move Object, String, Array and etc to `binding/JS` namespace instead `binding/DOM'? But rest of WebAPI objects (HTMLCanvasElement, CanvasRenderingContext2D, ...) keep here